discovery-provider/src/utils/helpers.py

```diff
@@ -27,6 +28,16 @@ def get_ip(request_obj):
     return ip.split(",")[0].strip()
 
 
+def get_openresty_public_key():
+    """Get public key for openresty if it is running"""
+    try:
+        resp = requests.get("http://localhost:5000/openresty_pubkey")
```

Comment:
this should never change after a container comes up right? and openresty should always boot up first before web server

Reply:
Yes, that's right